drivers/node/node_registry.go

```diff
@@ -206,3 +206,15 @@ func GetNodeByIP(nodeIP string) (Node, error) {
 func CleanupRegistry() {
 	nodeRegistry = make(map[string]Node)
 }
+
+// GetNodeDetailsByNodeName get node details for a given node name
+func GetNodeDetailsByNodeName(nodeName string) (Node, error) {
+	StorageNodes := GetStorageNodes()
+
+	for _, each := range StorageNodes {
+		if each.Name == nodeName {
+			return each, nil
+		}
+	}
+	return Node{}, fmt.Errorf("FAILED to get Node Details by Node Name [%s] ", nodeName)
```

Comment:
Please change it to `fmt.Errorf("failed to get node [%s] details", nodeName)` We don't need all in Capitals FAILED

Reply:
done